lansweeper_app_for_splunk/default/data/ui/views/ls_assets.xml

```diff
@@ -31,19 +31,34 @@
       <title>Asset States</title>
       <table>
         <search>
-          <query>`ls_assets_data` site_name=$tkn_site|s$ | dedup id | rename "assetCustom.stateName" as State | stats count by State</query>
+          <query>`ls_assets_data`  | eval id=coalesce(AssetID,id)| dedup id  | stats count by Statename</query>
           <earliest>-4h@h</earliest>
           <latest>now</latest>
         </search>
         <option name="drilldown">none</option>
         <option name="refresh.display">progressbar</option>
       </table>
     </panel>
+    <panel>
+      <title>Assets With Lansweeper Agent</title>
```

Comment:
This seems really useful. One suggestion though (because label going is bit complex, just to simplify it.):
- Let's rename it to "Lansweeper Agent", and rename the count column to AssetCount.

Reply:
Actually, I was not also satisfied but I was not able to think something else :). Made suggested changes.